src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -335,24 +336,28 @@ def assign_random_employer(
         )
 
     def assign_different_employer(self, changing_jobs: pd.Index) -> pd.Series:
-        # TODO: This limits employers to those existing in the pop table. If
-        # we want all employers that were initialized to be included, we can
-        # instead use the business data structure.
-        current_employers = (
-            self.population_view.subview(["employer_id"])
-            .get(changing_jobs, query="tracked")
-            .squeeze()
-        )
+        current_employers = self.population_view.subview(["employer_id"]).get(
+            changing_jobs, query="tracked"
+        )["employer_id"]
```

Comment:
Can do `squeeze(axis=1)` which I prefer.

Reply:
TIL! That is nicer.